apps/services/utils.py

```diff
@@ -119,3 +119,12 @@ def check_file_existing(path: str) -> None:
             message=f"File with path {path} was removed or deleted",
             code=http_status.HTTP_409_CONFLICT,
         )
+
+
+def check_user_request_status(status_id: int) -> None:
+    if status_id != 1:
```

Comment:
What does it mean? Maybe it's better to add some Enum, that will hold a `1` value, instead of a hardcoded value.

Reply:
fixed